app/views/project_mailer/project_created.html.slim

```diff
@@ -0,0 +1,6 @@
+h1
+  |  New project created.
+p
+  |  Congratulations! You have become the owner of a new project -- "
```

Comment:
You can use `'` instead of `|` for add space at the end of string line instead of using two spaces after `|`